### DISCUSSION TITLE
The tutorials in Nuxt.js site should be updated

### DISCUSSION THREAD

**Discussion by murugappanrm** [nuxt/nuxt#10362] [Questions (Nuxt 2)] [answered]
Hi,

The tutorials in the Nuxtjs website, like many others like Vuetify, are all outdated and should be removed. Please put in latest tutorials that work with the latest documentation.  Very embarrassing to note the software producer themselves do keep their sites updated.

**Comment by danielroe** [comment, marked as answer]
Pull requests are very welcome: https://github.com/nuxt/nuxtjs.org/ ❤️
